parse_emails/handle_msg.py

```diff
@@ -734,7 +736,11 @@ def _get_property_data(self, directory_name, directory_entry, is_list=False):
                          'skipping property "{}"'.format(stream_name, property_details))
             return None
 
+        utf_16_encoding = UTF_16_ENCODING
+        if directory_entry_name.endswith('001E'):
```

Comment:
Based on [this](https://github.com/Syncurity/msg-extractor/blob/master/extract_msg/dev_classes/message.py#L136) function
And [this](https://github.com/TeamMsgExtractor/msg-extractor/issues/71) issue